### PR TITLE
adding PATCH articles enpoint functions

### PR DESCRIPTION
**Pull request by JRGPower**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -225,6 +225,88 @@ describe('ENDPOINT TESTS', () => {
...
+                            body: expect.any(String),
+                            created_at: expect.any(String),
+                            votes: expect.any(Number),
+                        })
+                    );
+                    const dateCreated = new Date(res.body.article.created_at)
+                    expect(dateCreated).toBeInstanceOf(Date)
+                    expect(res.body.article.votes).toBeGreaterThanOrEqual(10)
+                });
+        });
```
Comment (sirKevlar) [inline __tests__/app.test.js:250]: It would be more thorough to add a test here for inc_votes being a negative number
